### PR TITLE
UPDARE login vue

### PR DESCRIPTION
**Pull request by Cmmeda**

### CODE REVIEW

File: pages/authen/login.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {{authStatus}}
    <form class="login" @submit.prevent="login">
      <h1>Sign in</h1>
-     <h1> tryyyy bb</h1>
+     <h1>Sign in</h1>
```
Comment (deal19890) [inline pages/authen/login.vue:6]: Duplicate Sign in

**Review by deal19890 — APPROVED** [review]
lgtm

File: pages/authen/login.vue (hunk 1 above)
Comment (rsxss) [reply pages/authen/login.vue:6]: Was a intentional duplicated

Comment (Cmmeda) [reply pages/authen/login.vue:6]: ok

**Pull request merged**